glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -78,6 +81,10 @@ open class GleanInternalAPI internal constructor () {
         this.gleanDataDir = File(applicationContext.applicationInfo.dataDir, GLEAN_DATA_DIR)
         handle = LibGleanFFI.INSTANCE.glean_initialize(this.gleanDataDir.path, applicationContext.packageName)
 
+        uploadEnabled?.let {
```

Comment:
This is not needed anymore, let's always pass the `uploadEnabled` status through the initialize method.